triedb/pathdb/states.go

```diff
@@ -98,6 +98,17 @@ func (s *stateSet) account(hash common.Hash) ([]byte, bool) {
 	return nil, false // account is unknown in this set
 }
 
+// mustAccount returns the account data associated with the specified address
+// hash. The difference is this function will return an error if the account
+// is not found.
```

Comment:
isn't it the wrong name then ? For example, we used to have `MustCommit` which would panic if it couldn't commit the block.

Reply:
Yeah maybe `tryAccount`?

Reply:
yeah, let me find a more reasonable name